### ISSUE TITLE
Foul spawn mage magic resist

### ISSUE THREAD

**Issue by oldmidgetwillow** [elmokki/nationgen#1213]
Foulspawn mages, especially those with multiple options, are spawning with sub-10 magic resist; I've seen as low as MR3, which is not particularly viable.

**Comment by Drithyl** [comment]
Is this still an issue in recent versions? I can't say that I've noticed it, but then I haven't been looking too closely.

**Comment by oldmidgetwillow** [comment]
This is still an issue, with primary mages having 8-10 MR or so.

**Comment by Drithyl** [comment]
It seems to simply be due to the possible bases and bodyparts. The animal related ones can start at base MR 8 and then have negatives as well, although MR 3 does surprise me. Others swing hard the other way; for example the "eye" head has +6 MR, so when it sits on top of a good base MR chassis it reaches levels like 22.

**Comment by oldmidgetwillow** [comment]
Yeah it's in a roughly viable place with the foul spawn fixes I put in--I gave a separate base for casters that roughly replicates the MR boost that was missing for montag casters. It's swingy, but you aren't automatically losing vs any astral or glamour nation anymore.

**Comment by Drithyl** [comment]
Closed for now as this is no longer an issue per se - just potential future rebalancing.
